internal/configv2/viperutils.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configv2
```

Comment:
Personally don't like calling things "v1" "v2". Are we planning to remove this once the old config is removed? If so, please file a issue.

Reply:
Yes, once V2 implementation is completed I plan a cleanup, where I remove unneeded V1 and rename configV2 simply to config.
I filed an issue to do that: https://github.com/census-instrumentation/opencensus-service/issues/546 (I am unable to self-assign this to me, please assign it).